src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+:root {
```

Comment:
The `:root` pseudo-class selector is used to select the root element of a document. In an HTML document, the root element is always the `html` element. It's a good practice to define your CSS variables here. Good job!

Reply:
And what  I need to fix in this case?